src/modules/mints/MintVolumeChart.tsx

```diff
@@ -0,0 +1,36 @@
+import moment from "moment";
+import { FunctionComponent } from "react";
+import { MintEntities } from "../../types/entities";
+import { BarChart } from "../../uikit/charts/BarChart";
+
+type MintVolumeChartProps = {
+	mintEntities: MintEntities[];
+};
+
+const MintVolumeChart: FunctionComponent<MintVolumeChartProps> = ({
+	mintEntities,
+}) => {
+	const mintAmountGroupByDate: { [day: string]: number } = {};
+	mintEntities.forEach((item) => {
+		const day: string = moment
+			.unix(parseInt(item.timestamp))
+			.format("YYYY-MM-DD");
+		const addedAmount: number = parseFloat(item.amountRISE) / 10 ** 18;
```

Comment:
it's better to create function about this
so people understand what's this calculation about